<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Audio input device</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Audio output device</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbAudioInputDevice"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cbAudioOutputDevice"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Bitrate</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Encoding type</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cbBitrate"/>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="cbEncodingType"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lbCurrentSec">
        <property name="text">
         <string>00:00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="slProgress">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbTotalSecs">
        <property name="text">
         <string>00:00</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btRec">
        <property name="text">
         <string>Rec</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btPlay">
        <property name="text">
         <string>Play</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btPause">
        <property name="text">
         <string>Pause</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btStop">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionSpecify_file_name"/>
    <addaction name="separator"/>
    <addaction name="action_Close"/>
   </widget>
   <widget class="QMenu" name="menuDevices">
    <property name="title">
     <string>&amp;Devices</string>
    </property>
    <addaction name="actionRefresh"/>
   </widget>
   <widget class="QMenu" name="menu_About">
    <property name="title">
     <string>&amp;About</string>
    </property>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuDevices"/>
   <addaction name="menu_About"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSpecify_file_name"/>
   <addaction name="separator"/>
   <addaction name="actionRefresh"/>
   <addaction name="separator"/>
   <addaction name="action_Close"/>
  </widget>
  <action name="action_Close">
   <property name="icon">
    <iconset resource="../../res/qcodec2_gui.qrc">
     <normaloff>:/icons/Close</normaloff>:/icons/Close</iconset>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="toolTip">
    <string>Close application</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="../../res/qcodec2_gui.qrc">
     <normaloff>:/icons/Refresh</normaloff>:/icons/Refresh</iconset>
   </property>
   <property name="text">
    <string>&amp;Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh audio devices list</string>
   </property>
  </action>
  <action name="actionSpecify_file_name">
   <property name="icon">
    <iconset resource="../../res/qcodec2_gui.qrc">
     <normaloff>:/icons/SpecifyFile</normaloff>:/icons/SpecifyFile</iconset>
   </property>
   <property name="text">
    <string>Specify file name...</string>
   </property>
   <property name="toolTip">
    <string>Specify file name</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../res/qcodec2_gui.qrc"/>
 </resources>
 <connections/>
</ui>
